src/base/bittorrent/session.cpp

```diff
@@ -346,6 +347,7 @@ Session::Session(QObject *parent)
     pack.set_int(libt::settings_pack::ssl_listen, 0);
     // To prevent ISPs from blocking seeding
     pack.set_bool(libt::settings_pack::lazy_bitfields, true);
+    pack.set_int(libt::settings_pack::alert_queue_size, 10000);
```

Comment:
I figured out that a torrent message could have a quite large memory footprint because the torrent handle is in there. I don't know which of the structures are just pointers to already allocated data an which not. But even if they are just pointers we would prevent libtorrent from releasing any memory.

Which mean it's better to invest the time for a messege fetch implementation during long running qbittorent thread functions or during triggering of many messages by qbittorent thread function.